cmd/lncli/commands.go

```diff
@@ -1481,7 +1481,7 @@ func capturePassword(instruction string, optional bool,
 			continue
```

Comment:
please reformat the commit message to something like:
"lncli: remove new line from password confirmation"

Also ptal: https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#Checklist